android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/components/TopScreenSection.kt

```diff
@@ -237,9 +237,7 @@ fun TopScreenSection(
             when {
               !searchQuery.isBlank() -> {
                 IconButton(
-                  onClick = {
-                    onSearchTextChanged(SearchQuery.emptyText, performSearchOnValueChanged)
-                  },
+                  onClick = { onSearchTextChanged(SearchQuery.emptyText, true) },
```

Comment:
Might it be better to fix it by [passing the value true from the line here](https://github.com/opensrp/fhircore/blob/main/android/quest/src/main/java/org/smartregister/fhircore/quest/ui/geowidget/GeoWidgetLauncherScreen.kt#L75)?

Or even consider removing it entirely since the value true is the default.

Reply:
@ndegwamartin We want to automatically disable search on text change for the map but enable it for the registers.  [This](https://github.com/opensrp/fhircore/blob/main/android/quest/src/main/java/org/smartregister/fhircore/quest/ui/geowidget/GeoWidgetLauncherScreen.kt#L75) disables it for the map. 

We however want to refresh the map view every time we clear data for both the register and map views hence the `true`.  If we use `performSearchOnValueChanged` this is updated to `false`  on the link you shared earlier.